static/js/services/plan-service.js

```diff
@@ -69,7 +69,8 @@ function(KeyService, UserService, CookieService, ApiService, Features, Config, $
         if (planService.isOrgCompatible(plan)) {
           $location.path('/organizations/new').search('plan', planId);
         } else {
-          $location.path('/user').search('plan', planId);
+          const username = UserService.currentUser().username;
+          $location.path(`/user/${username}/billing`);
```

Comment:
Just want to confirm if this the correct endpoint as I was not able to find a flask route for it.

Reply:
This isn't an API route. It is a frontend route. You can check the frontend route here `https://quay.io/user/{your_username}/billing`

Reply:
Oh! Got it. Thanks!